### ISSUE TITLE
Return to previous page button

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#2531]
Originally reported on Google Code with ID 1629

```
Please explain your idea here.
Hi I actually wanted to contribute to the project. I 'played' with the application
for a while and noticed that when you go from the starting page where you choose the
deck you want to view, and 'click' on the button at the bottom of the screen which
takes you to search for cards that you have installed(the second button from the left
with the 3 little rectangles), if you want to go back to the home page, you need to
press the back button of your phone.My opinion is that you should also go back to home
screen by pressing the icon of the Application at the top left corner of your screen(this
is for OS above Honeycomb).So I added that minor 'enhancement', I hope you will add
it to the App :-)
So basically I went to the CardBrowser.java and added a few lines of code at the OnCreate()
methode.
Cheers
Sokratis 

Is this feature available on similar apps (Anki Desktop, AnkiMobile,
AnyMemo, etc) ? If yes, it could help to include a screenshot or link to a
page explaining the feature.
This feature is NOT available in AnkiMobile.
He is a screenshot of what I did:
http://i49.tinypic.com/2posf10.png
Sorry for bad arrow drawing :-P
```

Reported by `sokras2008` on 2013-01-23 11:09:56

<hr>
- _Attachment: [CardBrowser.java](https://storage.googleapis.com/google-code-attachments/ankidroid/issue-1629/comment-0/CardBrowser.java)_
